recipes/tomlplusplus/all/conanfile.py

```diff
@@ -11,7 +11,7 @@ class TomlPlusPlusConan(ConanFile):
               "toml", "json", "header-only", "single-header")
     url = "https://github.com/conan-io/conan-center-index"
     license = "MIT"
-    settings = {"compiler"}
+    settings = ("compiler")
```

Comment:
Using only parentheses in python does nothing.
If you want to create a tuple, you need to add a comma.